rclcpp/test/executors/test_multi_threaded_executor.cpp

```diff
@@ -89,7 +89,7 @@ TEST_F(TestMultiThreadedExecutor, timer_over_take) {
         double diff = std::abs((now - last).nanoseconds()) / 1.0e9;
         last = now;
 
-        if (diff < PERIOD - TOLERANCE || diff > PERIOD + TOLERANCE) {
+        if (diff < PERIOD - TOLERANCE) {
           executor.cancel();
           ASSERT_GT(diff, PERIOD - TOLERANCE);
           ASSERT_LT(diff, PERIOD + TOLERANCE);
```

Comment:
This second ASSERT is no longer necessary and can be removed.

Reply:
Shouldn't it be the first one since the if statement will only execute if `diff` < `PERIOD - TOLERANCE`?
So only `ASSERT_LT(diff, PERIOD - TOLERANCE);`

Reply:
We want the assertion to fail, which is what the first assertion will do if `diff` < `PERIOD - TOLERANCE`. Unless I'm misunderstanding the test.

Reply:
timer __over__ take :) Should've inferred that.

Reply:
I'm not sure if you meant to remove the assert, but it's still there.

Reply:
Strange, pushed again. Ready for review now.